src/dataEditor/dataEditorClient.ts

```diff
@@ -459,11 +459,17 @@ export class DataEditorClient implements vscode.Disposable {
       case MessageCommand.undoChange:
         await undo(this.omegaSessionId)
         await this.sendChangesInfo()
+        this.panel.webview.postMessage({
```

Comment:
Question: I don't quite know enough to understand these additions, and to a newbie it seems like one is clearing changes right after the changes are sent, which is slightly confusing. Can you tell me more?

Reply:
Id be happy to elaborate but I'm not so sure these lines are related to clearing byte highlights.